### ISSUE TITLE
Facebook feature doesn't work on Firefox 38/Chrome

### ISSUE THREAD

**Issue by xalicex** [cryptocat/cryptocat#731]
Hi,

The facebook feature doesn't work on Firefox 38. It never start, a blank windows open and it loading endlessy. I'm on Ubuntu 14.04

On Chrome on Windows 8.1 but you can login but it never start.
On Firefox on Windows 8.1 it just log you on the facebook website

**Comment by Fluffgar** [comment]
Sounds like it's the same issue as: https://github.com/cryptocat/cryptocat/issues/720 
It's an ongoing problem. 

" kaepora commented 9 days ago [at time of this post]

Yes I should remove/fix this feature."

**Comment by HLLNSTN** [comment]
yep same problem for me. and this is with all add-ons turned off, third party cookies allowed, 'do not track' turned off, etc.
i would really like to use cryptocat, because of it's high recommendation from the EFF here: https://www.eff.org/secure-messaging-scorecard
also the UI is really lovely ! but if i can't link it to a facebook, i don't think my friend will like the hassle of using it

**Comment by t1n1t00n** [comment]
maybe this is the reason...
"alking of Facebook - as expected, they're not going to sit idly by and allow some 23-year-old upstart like Nadim Kobeissi mess with their ability to collect information about their users.
The social network has just announced that its Chat API/XMPP Services (relied upon by Cryptocat's new feature) will be shut down by April 30th 2015. In other words: goodbye Facebook Encrypted Chat."
https://grahamcluley.com/2014/05/encrypt-facebook-chat-cryptocat/

**Comment by HLLNSTN** [comment]
maybe twitter ?
